homeassistant/components/sensor/darksky.py

```diff
@@ -127,7 +130,9 @@
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Setup the Dark Sky sensor."""
     # Validate the configuration
-    if None in (hass.config.latitude, hass.config.longitude):
+    latitude = config.get(CONF_LATITUDE, hass.config.latitude)
```

Comment:
Will do, heads up on this.. code was taken from components\weather\openweathermap.py so this file should probably handle lat/long the same